pkg/infoschema/infoschema_v2.go

```diff
@@ -118,6 +127,9 @@ type Data struct {
 	// Stores only id, name and schemaVersion in memory.
 	schemaID2Name atomic.Pointer[btree.BTreeG[schemaIDName]]
 
+	// referredForeignKeys records all table's ReferredFKInfo.
+	referredForeignKeys atomic.Pointer[btree.BTreeG[*referredForeignKeyItem]]
```

Comment:
Why it is organized as 

```
{dbName, tableName, schemaVersion} => fkName1
{dbName, tableName, schemaVersion} => fkName2
{dbName, tableName, schemaVersion} => fkName3
```

rather than 

```
{dbName, tableName, schemaVersion} => fkName[]
```

Is the order of fkName important?

Reply:
The memory waste is higher because {dbName, tableName, schemaVersion} for each fkName in the fkName[]

Reply:
address in [c45a3a4](https://github.com/pingcap/tidb/pull/59422/commits/c45a3a4904a74cdd0b4b110b3c7a47899909a4ff)